File: Python/Day 5/mini_parking_system_test.py
import unittest
from mini_parking_system_function import *

class TestMiniParkingSystemFunction(unittest.TestCase):
	def testing_if_parking_slot_is_empty(self):
		expect = [1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
		actual = mini_parking_system_function.add_car(1)
		self.assertEquals(expect, actual)

File: Java/Day 5/MiniParkingSystemTest.java
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class MiniParkingSystemTest{
	
	MiniParkingSystemFunction miniParkingSystem ;

	@BeforeEach
	public void setUp(){
		MiniParkingSystemFunction miniParkingSystem = new MiniParkingSystemFunction();

	}

	@Test
	public void TestMiniParkingSystemFunction(){


	}

	@Test
	public void TestIfParkingACarChangesTheSlotToOccupied(){
		//MiniParkingSystemFunction miniParkingSystem = new MiniParkingSystemFunction();
		int result = miniParkingSystem.parkCar(1);
		assertEquals(result, 1);

	}
}